e2e-tests/src/localstack.rs

```diff
@@ -45,10 +45,10 @@ impl LocalStack {
     pub async fn setup_sqs(&self) -> color_eyre::Result<()> {
         let list_queues_output = self.sqs_client.list_queues().send().await?;
         let queue_urls = list_queues_output.queue_urls();
-        log::debug!("Found {} queues", queue_urls.len());
+        println!("Found {} queues", queue_urls.len());
```

Comment:
it's changed back to print because it's tests?

Reply:
yes